lib/Doctrine/CouchDB/CouchDBClient.php

```diff
@@ -807,6 +807,29 @@ public function ensureFullCommit()
             throw HTTPException::fromResponse($path, $response);
         }
 
+        return $response->body;
+    }
+
+    /**
+     * Bulk get.
+     *
+     * @param array $docs
+     *
+     * @throws HTTPException
+     *
+     * @return array
```

Comment:
The body of the response is an array? What type exactly?

Reply:
As described on www.com.behttps://docs.couchdb.org/en/stable/api/database/bulk-api.html#db-bulk-get

Reply:
Your link is broken

Reply:
https://docs.couchdb.org/en/stable/api/database/bulk-api.html#db-bulk-get

Reply:
Ok so it should also be 
```suggestion
     * @return array<int, array<string, string>>
```
correct?